src/main/scala/nl.knaw.dans.easy.validatebag/rules/sequence/package.scala

```diff
@@ -49,8 +48,10 @@ package object sequence extends DebugEnhancedLogging {
     failIfNotTrueWithMessage(uri.getScheme == "urn", "Is-Version-Of value must be a URN")
     failIfNotTrueWithMessage(uri.getSchemeSpecificPart.startsWith("uuid:"), "Is-Version-Of URN must be of subtype UUID")
     val Array(_, uuidStr) = uri.getSchemeSpecificPart.split(':')
-    failIfNotTrueWithMessage(uuidStr.length == uuidHexLength + uuidCanonicalNumberOfDashes, "UUID must be in canonical textual representation")
-    UUID.fromString(uuidStr)
+    uuidStr.toUUID.toTry match {
```

Comment:
How about `uuidStr.toUUID.toTry.getOrRecover(e => fail(e.getMessage))`.
We created that thing for this purpose... Please apply in all PRs.